src/main/webapp/style.css

```diff
@@ -7,3 +7,48 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.uploadButton {
```

Comment:
nit: CSS standard is to hyphenate class names, so this would be `upload-button` and below would be `see-memes-button`, etc. I really like the naming, though--very specific!